Build-a-Technical-Documentation-Page.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
Great! 

Also you forgot to put a CSS file. Please add it. 🏃

Reply:
> - [ ] Great review @jovasal👍 I would like to suggest that you phrase your comments in a more polite manner. Instead of giving orders, try to make suggestions. For example, instead of saying "You forgot to put a CSS file. Please add it," you could say "I noticed that there isn't a CSS file included. Would it be possible to add one?"😊
> 
> - [ ] Additionally, it's important to include an explanation, or a "why," for each requested change. This helps the student receiving the feedback to understand the reasoning behind the suggestion.😁
> 
> - [ ] Finally, try to make your comments clear and easy to understand. This will help ensure that the student receiving the feedback can easily follow your suggestions and make the appropriate changes. Keep up the good work!👍

Reply:
Fantastic job! I notice to you that you never use CSS in this project. 🏃  Would it be possible to add one?

If you use CSS in your code, you can build a beautiful web in your career. 🏉 

Kindly, you can see the following page where you can learn much more. This page is too easy to understand ❤️ and you can enjoy it:

https://www.w3schools.com/css/